views/index.jade

```diff
@@ -23,19 +23,21 @@ block content
     div.container
       div.row
         div.col-md-6.whatis
-          h2 Build unstoppable applications
+          h2 Welcome to the Ethereum open source community
```

Comment:
The unique thing for me about Ethereum is the community.  Everything else is easily copied.  Feature/benefit lists - 
 'decentralized', 'unstoppable', 'low fee transactions', 'fast' - those are all being copied by by other repos and websites ad infititum.   What they can't copy is the people - the foundation devs, MyEtherWallet, Consensys, academics like Emin, EEA.  What an incredible community to join and find a place in!